e2eTests/newUser.e2e.js

```diff
@@ -42,5 +42,20 @@ test('User can create new community and register', async t => {
   await t.expect(messageInput.exists).ok()
   await t.typeText(messageInput, 'Hello everyone')
   await t.pressKey('enter')
-  // TODO: assert that the message appears in general channel
+
+  // Sent message is visible on the messages' list
+  const messagesList = Selector('ul').withAttribute('id', 'messages-scroll')
+  const messagesGroup = messagesList.find('div').withAttribute('class', /-messageCard-/)
+  await t.expect(messagesGroup.exists).ok({ timeout: 10000 })
+  await t.expect(messagesGroup.count).eql(1)
+
+  const messageGroupContent = messagesGroup.find('p').withAttribute('class', /-message-/)
+  await t.expect(messageGroupContent.exists).ok()
+  await t.expect(messageGroupContent.textContent).eql('Hello\xa0everyone')
+
+  // Send second message, should appear in the same messages group
```

Comment:
We could but it would slow down the test by 300 seconds because that's what our grouping time limit is right now.

The message is displayed only after it's successfully added to db (we react to orbitdb `write` event https://github.com/orbitdb/orbit-db/blob/main/API.md#write). In other words - if message wasn't saved to db, user would not see it in zbaylite.